### DISCUSSION TITLE
Herschel infrared heater not detected in device type

### DISCUSSION THREAD

**Discussion by jackycheung1572** [make-all/tuya-local#1863] [General]
Hi,

I am trying to add a Herschel infrared heater; however, the list of device type in the drop down does not contain any Herschel infrared heater.

Herschel infrared heater is in the supported device list. 

I can add a Hydrotherm Dynamic/X8 without any issues and thus I am pretty sure my tuyalocal installation is working.

I am currently using localtuya to control those Herschel heater and they work. 

Did I miss something here? Or it is a bug?  Thanks.

**Comment by make-all** [comment]
Often devices report slightly different info with different settings.  It takes 2 or 3 users reporting their log messages before the config is working 100%
